core/src/main/java/org/apache/carbondata/core/datastore/impl/FileFactory.java

```diff
@@ -87,6 +87,10 @@ else if (path.startsWith(CarbonCommonConstants.VIEWFSURL_PREFIX)) {
     return FileType.LOCAL;
   }
 
+  public static CarbonFile getCarbonFile(String path) {
```

Comment:
I don't think it is better to change the existing caller methods, going forward user can use this if he needs it.